### DISCUSSION TITLE
Root Path

### DISCUSSION THREAD

**Discussion by tayloraleach** [cmintey/wishlist#163] [General]
I am trying to host an instance on my Digital Ocean droplet, but am having trouble running the app via docker. 

Attempting to proxy to mysite.com/wishlist but all of the assets are expected to be at the root. Is there any way for me to configure this?

Thank you!

**Comment by cmintey** [comment]
Hi there, nothing exists currently. I will look into it and see what needs to be done

**Comment by cmintey** [comment]
Unfortunately, it looks like this isn't currently possible with SvelteKit as the base URL needs to be specified at build time
